src/applications/check-in/travel-claim/pages/complete/TravelClaimSuccessAlert.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { getUniqueFacilies } from '../../../utils/appointment';
+import { formatList } from '../../../utils/formatters';
+
+const TravelClaimSuccessAlert = props => {
+  const { appointments } = props;
+  const facilities = getUniqueFacilies(appointments);
+
+  const { t } = useTranslation();
+
+  return (
+    <div className="vads-u-margin-y--4">
+      <va-alert
+        show-icon
+        data-testid="travel-pay-message"
+        status="success"
+        uswds
+      >
+        <h2 slot="headline">
+          {t('claim-submitted', { count: facilities.length })}
+        </h2>
+        <p
+          className="vads-u-margin-top--0"
+          data-testid="travel-pay-single-claim-submitted"
```

Comment:
this data-testid should use dynamic logic to be `travel-pay-single-claim-submitted` or `travel-pay-multi-claim-submitted`, then the unit test can operate based on that.

Reply:
Good idea!